### PR TITLE
[Part 4] Ruby: Additional resources section to comply with the style guide

### PR DESCRIPTION
**Pull request by nikitarevenco**
## Because

This PR is a part of series of PRs to change Additional resources section accross the entire curriculum to comply with the style guide

## This PR

Common fixes include:
- Changing additional resources section to be on the bottom of the page
- Change 'other' word to 'related' as per style guide
- Change '*' to '-' as per style guide

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by JoshDevHub** [comment]
I'll review this, but going to leave a couple comments first:
1. When you're making a change that isn't just correcting a one line type or wording problem on a page, *please* make an issue first. This is partially for your benefit. If someone decided to close this PR, then you will have spent a lot of time on something that doesn't get merged. And this had a solid chance of getting closed because of the diff size. Like I said, I'll review it, but other maintainers may think that even this is too big to review and ask you to break it up. Making an issue first helps clarify these things and prevent misunderstandings and lost work.
2. That said, I appreciate the effort and initiative to give these things some attention :smiley:

**Comment by nikitarevenco** [comment]
> I'll review this, but going to leave a couple comments first:
> 
> 1. When you're making a change that isn't just correcting a one line type or wording problem on a page, _please_ make an issue first. This is partially for your benefit. If someone decided to close this PR, then you will have spent a lot of time on something that doesn't get merged. And this had a solid chance of getting closed because of the diff size. Like I said, I'll review it, but other maintainers may think that even this is too big to review and ask you to break it up. Making an issue first helps clarify these things and prevent misunderstandings and lost work.
> 2. That said, I appreciate the effort and initiative to give these things some attention 😃

Alright, sorry for not making an issue first. I will definitely make sure to start off with an issue in the future before making a big change.

File: ruby/conclusion/project_ruby_final.md
Comment (JoshDevHub) [inline ruby/conclusion/project_ruby_final.md:26]: Every place you've copied and pasted this sentence from the Layout Style Guide has a small problem:
```
This section contains helpful links to related content. It isn't required, so consider it supplemental.
```
The `isn't` uses an incorrect apostrophe character and Markdownlint is complaining about it. I've put in a PR to fix this on the layout style guide here: #27319 

They also need to be fixed in this PR before merging.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Looks mostly great. Just one small problem with the additional resources content.

Also the section for `ruby/automated_testing` isn't currently a part of the curriculum on the site, so there's no real reason to make any changes to that content.

File: ruby/conclusion/project_ruby_final.md
Comment (nikitarevenco) [reply ruby/conclusion/project_ruby_final.md:26]: Well, this is awkward. I've copied and pasted from the layout style guide for **every** PR, which there are 7 of. 

I'll take time to fix all of the apostrophes in every PR, but in the future I'll definitely make issue before doing a big change like this as it wouldn't have happened had I made one!

Comment (nikitarevenco) [reply ruby/conclusion/project_ruby_final.md:26]: > Every place you've copied and pasted this sentence from the Layout Style Guide has a small problem:
> 
> ```
> This section contains helpful links to related content. It isn't required, so consider it supplemental.
> ```
> 
> The `isn't` uses an incorrect apostrophe character and Markdownlint is complaining about it. I've put in a PR to fix this on the layout style guide here: #27319
> 
> They also need to be fixed in this PR before merging.

I've converted all the ` to '. Reverting the 4 commits was tricky to do, since I've been using GitHub in the browser and hadn't had to clone the repository yet.

**Review by JoshDevHub — APPROVED** [review]
Looks good. Thanks for doing this.

**Pull request merged**
